src/reducers/app/thunks.ts

```diff
@@ -157,12 +157,12 @@ export const submitQuery = () => async (dispatch, state) => {
   });
   // query = setLimit(query, limit);
   // Add this to set ordering
+  alert(attributeNames);
   query = addOrdering(
     query,
-    attributeNames.includes('block_level') ? 'block_level' : 'level',
+    !attributeNames.includes('level') ? 'block_level' : 'level',
```

Comment:
These names need to be normalized at the Conseil service level

Reply:
Agreed. We will go over operators and attributes in another ticket